src/OpenLoco/src/GameState.h

```diff
@@ -61,7 +61,7 @@ namespace OpenLoco
         uint8_t lastLandOption;                                                  // 0x00019E (0x00525FB6)
         uint8_t maxCompetingCompanies;                                           // 0x00019F (0x00525FB7)
         uint32_t numOrders;                                                      // 0x0001A0 (0x00525FB8)
-        uint32_t var_1A4;                                                        // 0x0001A4 (0x00525FBC)
+        uint32_t roadObjectIdBits;                                               // 0x0001A4 (0x00525FBC)
```

Comment:
`roadObjectIdIsTram`  I know its not written anywhere but that is what this is for. Its a faster way of working out if a road object is a tram item.

Reply:
Great thanks for the clarification.
I'll rename to roadObjectIdIsTram.
I got roadObjectIdBits from a comment.